src/python/artm/library.py

```diff
@@ -564,7 +611,7 @@ def GetRegularizerState(self, regularizer_name):
         regularizer_state.ParseFromString(state_blob)
         return regularizer_state
 
-    def GetThetaMatrix(self, model=None, batch=None, clean_cache=None, args=None, topic_names=None):
+    def GetThetaMatrix(self, model=None, batch=None, clean_cache=None, args=None, topic_names=None, use_matrix=True):
```

Comment:
I have the only small question: why use_matrix default is True? It seems to broke the backward compatibility, doesn't it? False value is safe and if user need to speed up his code in low-level API, he will set it to True; the user of ArtmModel will has this option True by default in corresponding method implementation.